app/src/main/java/com/kamron/pogoiv/updater/DownloadUpdateService.java

```diff
@@ -36,7 +36,7 @@ public int onStartCommand(Intent intent, int flags, int startId) {
                 newApkFile.delete();
             }
             DownloadManager.Request request = new DownloadManager.Request(Uri.parse(downloadUrl));
-            request.setTitle(DOWNLOAD_UPDATE_TITLE);
+            request.setTitle(getBaseContext().getString(R.string.notification_updating));
```

Comment:
_getBaseContext()_ is useless, you could just use _getString(R.string.notification_updating)_ since the Service class extends Context